frontend/packages/core-api-client/src/client/Recipient.ts

```diff
@@ -21,17 +31,66 @@ export class RecipientClient {
     });
   };
 
-  list = (args: FormLookup): Promise<RecipientList> => {
+  list = (): Promise<RecipientList> => {
     return Promise.resolve({ items: [] });
   };
 
-  get = async (args: RecordLookup): Promise<Recipient> => {
-    return Promise.resolve({
-      id: 'id',
-      formId: 'formId',
-      databaseId: 'databaseId',
-      ownerId: undefined,
-      values: [],
+  get = (args: RecordLookup): Promise<RecipientResponse> => {
```

Comment:
Why have you changed the response type of this? I think we want to keep it as `Promise<Recipient>`

Reply:
moving the ancestor code into the get solved this though